modules/gumgumBidAdapter.js

```diff
@@ -51,31 +67,33 @@ function _getBrowserParams(topWindowUrl) {
     topUrl = topWindowUrl || '';
   } catch (error) {
     logError(error);
-    return browserParams
+    return browserParams;
   }
 
   browserParams = {
     vw: topWindow.innerWidth,
     vh: topWindow.innerHeight,
     sw: topScreen.width,
     sh: topScreen.height,
-    pu: topUrl,
+    pu: stripGGParams(topUrl),
     ce: storage.cookiesAreEnabled(),
     dpr: topWindow.devicePixelRatio || 1,
     jcsi: JSON.stringify(JCSI),
     ogu: getOgURL()
-  }
+  };
 
-  ns = getNetworkSpeed()
+  ns = getNetworkSpeed();
   if (ns) {
-    browserParams.ns = ns
+    browserParams.ns = ns;
   }
 
-  ggad = (topUrl.match(/#ggad=(\w+)$/) || [0, 0])[1]
-  if (ggad) {
-    browserParams[isNaN(ggad) ? 'eAdBuyId' : 'adBuyId'] = ggad
-  }
-  return browserParams
+  ggad = (topUrl.match(paramRegex('ggad')) || [0, 0, 0])[2];
```

Comment:
Same question about [2]. Not sure how it is with public repos but comments may be helpful to indicate what these magic numbers are and why we use them like this.

Reply:
so (as per above comment response lol), 2 is the value of the parameter if there's a match, otherwise, it's the third value in that array of 0's (where the first two values could be anything but ¯\_(ツ)_/¯)